messaging.rst

```diff
@@ -504,19 +504,27 @@ Preconditions
 - The message sender address must be the same as ``participant``.
 - The ``signature`` must be from the :term:`sender` of the request.
 
-Fields
-^^^^^^
+Fields and signature
+^^^^^^^^^^^^^^^^^^^^
+
+``WithdrawRequest`` is an :term:`onchain message`. This is the format in which it is packed to
+compute the ``signature``, and in which it could be forwarded to a contract.
```

Comment:
The `WithdrawRequest` itself is never sent to the blockchain, but its signature will be used (along with the signature of the `WithdrawConfirmation`) to withdraw tokens by calling `setTotalWithdraw` on the token network contract.